### ISSUE TITLE
Downloaded dependencies should not be installed

### ISSUE THREAD

**Issue by awvwgk** [qcscine/cmake#3]
This is a common issue with nested CMake projects (also happens in other build systems like meson), when pulling in third-party dependencies the third party-dependency should better not be installed by default.

In this case I build scine-core with

```
cmake -GNinja -B_build -S. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=/usr
DESTDIR=$PWD/install ninja -C _build all install
```

This could be a potential piece of a conda-build recipe or an AUR `PKGBUILD` instruction, which could produce a binary distribution at some point. As mentioned in #1 the test dependencies should not be necessary for the final installation. Unfortunately, I'm not getting only the scine-core installed (with `libcore.so` being an interesting and potential risky choice of name) but also a stray `gtest` installation:

```
[0/1] Install the project...
-- Install configuration: "Release"
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/Scine/ScineConfig.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-matchers.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-nice-strict.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-matchers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-spec-builders.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-more-matchers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-function-mockers.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom/gmock-port.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom/gmock-generated-actions.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom/gmock-generated-actions.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom/README.md
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/custom/gmock-matchers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/gmock-generated-internal-utils.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/gmock-port.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/gmock-generated-internal-utils.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/internal/gmock-internal-utils.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-cardinalities.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-actions.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-actions.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-nice-strict.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-generated-function-mockers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-actions.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-more-actions.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock-matchers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gmock/gmock.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/libgmock.a
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/libgmock_main.a
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/pkgconfig/gmock.pc
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/pkgconfig/gmock_main.pc
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/GTest/GTestTargets.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/GTest/GTestTargets-release.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/GTest/GTestConfigVersion.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/GTest/GTestConfig.cmake
-- Up-to-date: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-message.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-typed-test.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest_prod.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-spi.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-string.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-type-util.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-tuple.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/custom
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/custom/gtest-port.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/custom/gtest.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/custom/README.md
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/custom/gtest-printers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-port.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-param-util.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-death-test-internal.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-type-util.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-filepath.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-tuple.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-internal.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-port-arch.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-linked_ptr.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-param-util-generated.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/internal/gtest-param-util-generated.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-test-part.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-death-test.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-printers.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-param-test.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest-param-test.h.pump
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/gtest/gtest_pred_impl.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/libgtest.a
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/libgtest_main.a
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/pkgconfig/gtest.pc
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/pkgconfig/gtest_main.pc
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/ModuleManager.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Module.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/BaseClasses
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/BaseClasses/StateHandableObject.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/BaseClasses/ObjectWithStructure.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Exceptions.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/ExportControl.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Interfaces
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Interfaces/CalculatorWithReference.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Interfaces/MMParametrizer.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Interfaces/Calculator.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/Interfaces/WavefunctionOutputGenerator.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/include/Core/DerivedModule.h
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/libcore.so
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/Scine/Core/CoreConfig.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/Scine/Core/CoreConfigVersion.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/Scine/Core/CoreTargets.cmake
-- Installing: /home/awvwgk/projects/src/github.com/scine/core/install/usr/lib/cmake/Scine/Core/CoreTargets-release.cmake
```

**Comment by nabbelbabbel** [comment]
Indeed it should not be installed, another fix that is already staged for the next release. I would ask you to wait a few weeks, most of these issues will be resolved as there will probably be a binary distribution then.

**Comment by weymutht** [comment]
This has been fixed with release 4.0.0 of development-utils. Therefore, I close this issue.
